code/game/machinery/suit_storage_unit.dm

```diff
@@ -211,13 +212,6 @@
 	storage = null
 	set_occupant(null)
 
-/obj/machinery/suit_storage_unit/deconstruct(disassembled = TRUE)
```

Comment:
shouldnt this stay so that it drops everything on deconstruction still? just change the drop/qdel?

Reply:
will look at this, busy atm